### DISCUSSION TITLE
Canadian death data show a one-day large increase

### DISCUSSION THREAD

**Discussion by marcopiani** [owid/covid-19-data#2698] [General]
Hi,

The Canadian data show a one-day large increase for deaths. This has happened many times for many countries, due to data reconciliations etc.
![FX3dg_0WAAI66Fm](https://user-images.githubusercontent.com/18381804/179398189-fc882ae7-b595-4f2c-a104-2e9f665b6f95.png)

Wondering whether 1) this could be confirmed and in case 2) "adjusted" in some way via some spreading of the large increase over time, perhaps in collaboration with JHU.

No particular need to do this specifically in this case, given that it is a relatively common occurrence and especially if it is complicated / time consuming. Just wondering.

Thank you!

Marco

**Comment by lucasrodes** [comment]
Hi @marcopiani,
Thanks for reporting.

There might be an issue with Ontario's data (see https://github.com/CSSEGISandData/COVID-19/issues/5892).

Let's wait to see if this gets corrected. We haven't planned a collaboration to fix these data points, but we might explore it. Thanks for proposing it!

**Reply by marcopiani** [reply]
Thank you!

**Comment by lucasrodes** [comment]
This has now been resolved in https://github.com/CSSEGISandData/COVID-19/issues/5892?notification_referrer_id=NT_kwDOARQ0KbM0MDA3NDYxMTI4OjE4MTAxMjg5#issuecomment-1196417988, @marcopiani!

**Reply by marcopiani** [reply]
Thank you for your efforts!
